Tasks/DockerV2/Tests/TestShared.ts

```diff
@@ -9,7 +9,8 @@ export let TestEnvVars = {
     tags: "__tags__",
     arguments: "__arguments__",
     container: "__container__",
-    addPipelineData: "__addPipelineData__"
+    addPipelineData: "__addPipelineData__",
+    addBaseImageData: "addBaseImageData"
```

Comment:
should this env variable follow the format of the rest using the underscores?

Reply:
yeah..lets stick to the existing format.